src/components/Sections/SectionView.tsx

```diff
@@ -16,41 +16,52 @@ import {
   SectionViewProps
 } from 'components/Sections/SectionsTypes'
 import { useSections } from './SectionsContext'
+import CandidateServiceTile from 'components/CandidateServiceTile'
 
 export const SectionBody = ({ section }: SectionViewProps): JSX.Element => {
   const { isMobile } = useBreakpoints()
   const currentDisplayMode = computeDisplayMode(isMobile, section)
-  const isGroupMode =
-    (section && computeGroupMode(isMobile, section)) === GroupMode.GROUPED
+  const isGroupMode = computeGroupMode(isMobile, section) === GroupMode.GROUPED
   const { t } = useI18n()
-  const shouldOpenStoreModal = Boolean(
-    section.type === 'category' && section.pristine
-  )
-  const { isRunning } = useSections()
+  const shouldOpenStoreModal = section.type === 'category' && section.pristine
+  const { isRunning, isInMaintenance } = useSections()
 
   return (
     <div
       className={cx(
-        'shortcuts-list u-w-100 ',
+        'shortcuts-list u-w-100',
         {
           'u-mv-3 u-mv-2-t u-mh-auto u-flex-justify-center': !isGroupMode
         },
-        { detailed: Boolean(currentDisplayMode === DisplayMode.DETAILED) }
+        { detailed: currentDisplayMode === DisplayMode.DETAILED }
       )}
     >
-      {(section.items as IOCozyKonnector[]).map((item, index) =>
-        item.type === 'konnector' ? (
-          <KonnectorTile
-            shouldOpenStore={shouldOpenStoreModal}
-            key={item.slug}
-            konnector={item}
-            isInMaintenance={false}
-            loading={isRunning(item.slug)}
-          />
-        ) : (
-          <ShortcutLink key={index} file={item} display={currentDisplayMode} />
-        )
-      )}
+      {(section.items as IOCozyKonnector[]).map((item, index) => {
```

Comment:
This typing can lead to errors because items can be both konnectors and suggestions. We could add `IOCozySuggestions` inside cozy-client to improve this in second time and use type from Section ([code](https://github.com/cozy/cozy-home/blob/a4ca9022564579be32b673ee6080c059af914e29/src/components/Sections/SectionsTypes.ts#L13)) without the cast